lib/Runtime/ByteCode/FuncInfo.cpp

```diff
@@ -136,22 +136,21 @@ BOOL FuncInfo::HasDirectSuper() const
 
 BOOL FuncInfo::IsClassMember() const
 {
-    return root->sxFnc.IsClassMember();
+    return this->byteCodeFunction->IsClassMethod();
 }
 
-BOOL FuncInfo::IsLambda() const
-{
-    return root->sxFnc.IsLambda();
+BOOL FuncInfo::IsLambda() const {
```

Comment:
Nit (sorry), please keep curly style consistent.

Reply:
Ah, some style crept-in from JavaScript. I'll clean this one.